packages/turf-nearest-point-to-line/index.js

```diff
@@ -0,0 +1,45 @@
+var meta = require('@turf/meta');
+var invariant = require('@turf/invariant');
+var pointToLineDistance = require('@turf/point-to-line-distance');
+var featureOf = invariant.featureOf;
+var featureEach = meta.featureEach;
+var collectionOf = invariant.collectionOf;
+
+/**
+ * Returns the closest {@link Point|point}, of a {@link FeatureCollection|collection} of points, to a {@link LineString|line}.
+ * The returned point has a `dist` property indicating its distance to the line.
+ *
+ * @name nearestPointToLine
+ * @param {FeatureCollection<Point>} points collection
+ * @param {Feature<LineString>} line Feature
+ * @param {string} [units=kilometers] can be degrees, radians, miles, or kilometers
```

Comment:
> @param {string} [units=kilometers] can be degrees, radians, miles, or kilometer

My only comment would be saying that this is for the output property. So perhaps
`@param {string} [units=kilometers] unit of the output distance property, can be degrees, radians, miles, or kilometer`